pages/en/network/developing.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Developing
+---
+
+Developers are a core part of The Graph. They are responsible for both building and querying subgraphs, unlocking data for the web3 ecosystem.
+
+## Building Subgraphs
+
+Developers that build subgraphs use the [GraphQL Schema Language](https://graphql.org/learn/schema-language/) to define the data they want to make available.
+
+The Graph CLI tool streamlines subgraph creation. When you initialize your subgraph, all the default files will be created & configured. Developers can use the built-in APIs or add other libraries as dependencies as long as they are compatible with [AssemblyScript](https://github.com/AssemblyScript/assemblyscript). Subgraphs should be thoroughly tested before deployment. Developers can use [MatchStick](https://github.com/LimeChain/matchstick) to test the mapping logic in a sandboxed environment. Once your subgraph is built & tested, it can be deployed to The Graph's decentralized network. However, before anyone can begin querying it, your subgraph must be curated & indexed.
+
+### Curation
+
+Curators use their knowledge of web3 to assess and signal new subgraphs. They look for subgraphs with reliable and relevant data.
+
+Curators stake GRT with the subgraph and receive curation shares, which guarantee a percentage of future query fees. The curation signal increases as more Curators signal on the subgraph.
+
+A strong curation signal is crucial to a subgraph. Indexers use the curation signal to determine which subgraphs to index to the network, making it available for querying.
+
+### Deploying on the network
+
+By deploying your subgraph to The Graph network, you're making that data available to developers and users across the web3 ecosystem. Anyone will be able to query it.
+
+Data from your subgraph can be used to improve user experience, build new features, or even create entirely new applications.
+
+## Querying Subgraphs
+
+Developers that query subgraphs serve it to users through a simple interface. They can use the GraphQL API to query data directly from the blockchain. Developers can manipulate, sort, or aggregate the data any way they want.
```

Comment:
When you query a subgraph, you're not querying data directly from the blockchain, you are querying it from the subgraph's processed database

Reply:
Good point, correcting that